frameworks/PHP/symfony/deploy/nginx.conf

```diff
@@ -1,37 +1,37 @@
-#user  nobody;
 worker_processes  8;
 
-#error_log  logs/error.log;
-#error_log  logs/error.log  notice;
-#error_log  logs/error.log  info;
-error_log stderr error;
-
-#pid        logs/nginx.pid;
-
-
 events {
-    worker_connections  1024;
+    worker_connections 2048;
+    multi_accept on;
+    use epoll;
 }
 
-
 http {
-    include       /usr/local/nginx/conf/mime.types;
-    default_type  application/octet-stream;
-
-    #log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
-    #                  '$status $body_bytes_sent "$http_referer" '
-    #                  '"$http_user_agent" "$http_x_forwarded_for"';
+    include /home/vagrant/FrameworkBenchmarks/installs/nginx/conf/mime.types;
```

Comment:
If you're adding in these lines you'll have to fix them in the setup files like laravel is doing here: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/laravel/setup.sh#L7